extensions/controllers/sandboxclaim_controller.go

```diff
@@ -255,6 +255,8 @@ func (r *SandboxClaimReconciler) createSandbox(ctx context.Context, claim *exten
 		},
 	}
 	sandbox.Spec.PodTemplate = template.Spec.PodTemplate
+	replicas := int32(1)
+	sandbox.Spec.Replicas = &replicas
```

Comment:
This was needed because PDB needs to know how many replicas are available

Reply:
please add a TODO referring to #202